### PR TITLE
[Fix] fix bug of non-COCO dataset track demo

### PR DESCRIPTION
**Pull request by daixinghome**
## Motivation

Fix: https://github.com/open-mmlab/mmpose/issues/1502

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Comment by Ben-Louis** [comment]
@daixinghome Thank you very much for your contribution! Could you please sign the CLA?

**Review by jin-s13 — APPROVED** [review]

**Comment by daixinghome** [comment]
>

**Comment by daixinghome** [comment]
Sorry for the mistake. Now it's ok~

File: mmpose/apis/inference_tracking.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def _compute_iou(bboxA, bboxB):
     return iou
 
 
-def _track_by_iou(res, results_last, thr):
+def _track_by_iou(res, results_last, thr, **kwargs):
```
Comment (ly015) [inline mmpose/apis/inference_tracking.py:39]: I would recommend not using ``**kwargs`` but it seems there is no other simple solution here. @jin-s13 Do you have any suggestions?

Comment (Ben-Louis) [reply mmpose/apis/inference_tracking.py:39]: Maybe we can use `functools.partial` to unify the interfaces

Comment (ly015) [reply mmpose/apis/inference_tracking.py:39]: > Maybe we can use `functools.partial` to unify the interfaces

Could you please give an example?

Comment (Ben-Louis) [reply mmpose/apis/inference_tracking.py:39]: > Could you please give an example?

```python
def _track_by_iou(res, results_last, thr): 
    # track by iou

def _track_by_oks(res, results_last, thr, sigmas): 
    # track by oks

_track = partial(_track_by_oks, sigmas=dataset_sigmas) if use_oks else _track_by_iou
```

if `use_oks` is True, then `_track` acts like the following function
```python
def func(res, results_last, thr): 
    return _track_by_oks(res, results_last, thr, dataset_sigmas)
```

Comment (daixinghome) [reply mmpose/apis/inference_tracking.py]: Good idea. I've already modified with this method :).

**Pull request merged**